doc/source/guide/acquiring_data/fido.rst

```diff
@@ -0,0 +1,71 @@
+---------------------------------
+Finding and Downloading Data using Fido
+---------------------------------
+
+This guide outlines how to search for and download data using SunPy's
+Federated Internet Data Obtainer...or more usually (and
+sanely) referred to as Fido.  Fido is a unified interface for seeking
+and downloading solar physics data irrespective of the underlining
+client or webservice through which the data is obtained, e.g. VSO,
+JSOC etc.  It therefore supplies a single easy and consistent way to
+obtain most forms of solar physics data.
+
+Import
+-----
+
+SunPy's Fido module is in ``sunpy.net``.  It can be imported as follows::
+
+    >>> from sunpy.net import Fido, attrs as a
```

Comment:
I had this discussion in my head, I think the minor PEP8 violation of single character variable name is a sacrifice that's worth paying for a namespace that you use many times (like astropy.units)